WcaOnRails/db/migrate/20170316030443_add_big_json_to_competitions.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class AddBigJsonToCompetitions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :Competitions, :big_json, :text
```

Comment:
Without knowing much, I have these questions based on ignorance:

1. Do we want to store this in the DB? As opposed to generating them when needed. Cache invalidation is the main problem I worry about.
2. If we store it in the DB, I'd feel better having a separate table, so it doesn't get loaded each time a Competition is loaded. This is based in part on that the name makes it sound like it's a lot of data.